guidelines/index.html

```diff
@@ -3667,11 +3667,13 @@ <h2 id="x6-glossary"><span class="secno">6. </span>用語集<span class="permali
       
    </ul>
    
-   <p>^ という記号は、指数演算子である(計算式は、[<cite><a class="bibref" href="#bib-sRGB">sRGB</a></cite>] 及び [<cite><a class="bibref" href="#bib-IEC-4WD">IEC-4WD</a></cite>]を参考にしている)。
+   <p>^ という記号は、指数演算子である(計算式は、[<cite><a class="bibref" data-link-type="biblio" href="#bib-srgb" title="Multimedia systems and equipment - Colour measurement and management - Part 2-1: Colour management - Default RGB colour space - sRGB">SRGB</a></cite>] を参考にしている)。
    </p>
 
   </div></div>
-  
+
+  <div class="note" role="note" id="issue-container-generatedID-120"><div role="heading" class="note-title marker" id="h-note-120" aria-level="3"><span>注記</span></div><p class="">2021 年 5 月以前は、定義にある 0.04045 の値が異なっていた (0.03928)。これは、仕様の古いバージョンから取られたものであり、更新されている。本ガイドラインの文脈における計算には、実用上の影響はない。</p></div>
```

Comment:
旧版の仕様から取り込み、元の仕様は更新されている、というようなニュアンスだと思いますので、少し手を入れてみました。


```suggestion
  <div class="note" role="note" id="issue-container-generatedID-120"><div role="heading" class="note-title marker" id="h-note-120" aria-level="3"><span>注記</span></div><p class="">2021 年 5 月以前は、定義にある 0.04045 の値が異なっていた (0.03928)。これは、古いバージョンの仕様から取り込んだものであり、現在は更新されている。本ガイドラインの文脈における計算には、実質的な影響はない。</p></div>
```